galley/pkg/config/analysis/diag/message.go

```diff
@@ -63,6 +65,7 @@ func (m *Message) Unstructured(includeOrigin bool) map[string]interface{} {
 		result["origin"] = m.Origin.FriendlyName()
 	}
 	result["message"] = fmt.Sprintf(m.Type.Template(), m.Parameters...)
+	result["doc"] = fmt.Sprintf("%s/%s", DocPrefix, m.Type.Code())
```

Comment:
Re: https://github.com/istio/api/blob/master/GUIDELINES.md#placement and https://docs.google.com/document/d/1lCnDBVzRzkr22BWjzPtGk6EK6EvGOnVIe2qBpaiQ3SM/edit# can we make this "documentation_url" instead of "doc"?

Reply:
Done.

Reply:
Done